js/widgets/forms/flipswitch.js

```diff
@@ -251,9 +245,8 @@ return $.widget( "mobile.flipswitch", $.extend( {
 			var currentTheme = options.theme ? options.theme : "inherit",
 				newTheme = options.theme ? options.theme : "inherit";
 
-			this.widget()
-				.removeClass( "ui-bar-" + currentTheme )
-				.addClass( "ui-bar-" + newTheme );
+			this._removeClass( this.flipswitch, null,  "ui-bar-" + currentTheme );
+			this._addClass( this.flipswitch, null,  "ui-bar-" + newTheme );
```

Comment:
ok this is not new but this makes no sense `currentTheme` and `newTheme` cant possibly be different above so you are just removing and then adding the same class again.